protocol/0043-MKTL-market_lifecycle.md

```diff
@@ -293,3 +294,13 @@ Margin account balances are transferred to the general account.
 Any insurance pool balance is transferred to the network treasury account for the asset. 
 The market state is `settled`. 
 
+### Market never leaves opening auction, trading terminated trigger rings, market cancelled (<a name="0043-MKTL-003" href="#0043-MKTL-003">0043-MKTL-003</a>)
+
+1. A market is proposed, approved by governace process and enterst the opening auction (Pending state).
+1. Trading terminated data source rings before the market leaves the opening auction (so market never left Pending state so far).
+1. All orders should be cancelled and collateral returned to respective parties general account for the relevant asset.
+1. All LP commitments should be cancelled and their bond returned to the general account for the relevant asset.
+1. Any insurance pool balance should be transferred into the network treasury account for that asset. 
```

Comment:
not sure this applies. If market never leaved initial Pending state, there is no way we can have anything in the insurance pool (no positions, no trading, no distress, etc)

Reply:
In the future it will be possible to fund insurance pool with transfers. 

But yes, at this point you won't be able to test this.